src/PSRule/Configuration/OutputOption.cs

```diff
@@ -196,6 +209,9 @@ internal void Load(EnvironmentHelper env)
 
             if (env.TryEnum("PSRULE_OUTPUT_STYLE", out OutputStyle style))
                 Style = style;
+
+            if (env.TryInt("PSRULE_OUTPUT_JSON_INDENT", out int jsonIndent))
```

Comment:
Looks better :) but would like to keep consistency with `PSRULE_<key>_<key>`. So just `PSRULE_OUTPUT_JSONINDENT` is fine.